flight/drone/drone_controller.py

```diff
@@ -32,18 +34,23 @@ class DroneController(object):
         A PriorityQueue holding tasks to be performed.
     _safety_event : Event
         Set when an unsafe condition is observed.
+    _is_simulation : bool
+        Set to true when the code is intended for the simulator.
     """
 
-    def __init__(self, drone):
+    def __init__(self, drone, is_simulation=False):
         """Construct a drone controller.
 
         Parameters
         ----------
-        drone : c.Drone.{DRONE_NAME}"""
+        drone : c.Drone.{DRONE_NAME}
```

Comment:
Undocumented parameter is_simulation (also make sure to document the default value for this parameter)